cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -313,6 +365,33 @@ def get_kube_api_server(kube_api):
     return server
 
 
+def kubectl(operation, manifest):
+    ''' Wrap the kubectl creation command when using filepath resources
+    :param operation - one of get, create, delete, replace
+    :param manifest - filepath to the manifest
+     '''
+    kubectl = ['kubectl', '--kubeconfig=/srv/kubernetes/config']
+    # determine if the kubernetes resources have been declared already
+    found = call(kubectl + ['get', '-f', manifest])
```

Comment:
Calling get every time may be wasteful when operation is get, delete or replace. I only used get to prevent accidentally trying to create a new object of the same time as that would error out.